docs/full-configuration.md

```diff
@@ -50,12 +50,16 @@ If you don't configure any private host key, the daemon will use `id_rsa`, `id_e
 
 The `gen` command allows to generate completion scripts for your shell and man pages.
 
+## Configuration file
```

Comment:
why we need this and the other similar headings?

Reply:
I thought it was prettier with it than without. Especially at the end of the configuration segment to indicate that the sub segment has ended and that we are now once again in the upper segment.